AppBuilder/platform/ABModel.js

```diff
@@ -283,6 +283,19 @@ module.exports = class ABModel extends ABModelCore {
    async findAll(cond) {
       cond = cond || {};
 
+      let rules = cond.where.rules;
```

Comment:
Do me a favor and simply put in some comments describing what you are trying to do here. Something like:
```
// scan the rules and convert any is_current_date rules to have our date values
``` 

would be fine.